src/appmixer/resend/auth.js

```
@@ -0,0 +1,47 @@
+        requestProfileInfo: async (context) => {
+            const { data } = await context.httpRequest({
+                method: 'GET',
+                url: 'https://api.resend.com/v1/domains',
+                headers: {
+                    'Authorization': `Bearer ${context.apiKey}`
+                }
+            });
+            
+            // Use the first domain's email or a default
+            return {
+                email: data && data.data && data.data.length > 0
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)